test/test_rosbag/test/test_bag.py

```diff
@@ -312,6 +312,14 @@ def test_get_time(self):
             self.assertEquals(start_stamp, 0.0)
             self.assertEquals(end_stamp, 99.0)
 
+    def test_get_time_empty_bag(self):
+        """Test for issue #657"""
+        fn = '/tmp/test_get_time_emtpy_bag.bag'
```

Comment:
It should use a unique path though. [tempfile](https://docs.python.org/2/library/tempfile.html) provides methods for creating temporary directories properly. 

You can either generate unique filenames in each test or setup a directory for all the tests.